docs/sources/developers/plugins/publish-a-plugin.md

```diff
@@ -1,148 +1,44 @@
 ---
 aliases:
   - share-a-plugin/
-  - package-a-plugin/
 description: Learn how to package and share your plugin.
 title: Publish a plugin
 type: docs
 ---
 
-# Publish a plugin
+# Publish or update a plugin
 
-You've just built your first plugin, and now you want to share it with the world.
-In this guide, you'll learn how to package and share your plugin with others.
+You've just built your first plugin and [packaged it]({{< relref "package-a-plugin.md" >}}), and now you want to share it with the world.
```

Comment:
do we want to link "just built" to `_index.md#tutorials`?

Reply:
Added link to "/create-a-grafana-plugin/"